spec/support/bypass_fake_gitis.rb

```diff
@@ -0,0 +1,4 @@
+shared_context "bypass fake Gitis" do
+  before { Rails.application.config.x.fake_crm = false }
+  after { Rails.application.config.x.fake_crm = true }
```

Comment:
Super-minor, but is actually toggling a requirement here?

Would `before { allow(Rails.application.config.x).to receive(:fake_crm).and_return(false) }` work?

Reply:
Yep - really good suggestion! I'll get it changed to that